modules/ROOT/pages/getting-started-proxy.adoc

```diff
@@ -1,43 +1,47 @@
-= Quick start: Manage an API and set up a proxy for it
+= Getting Started with Managing an API 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this walkthrough, we are showing how to create an API that can be managed through API Manager and set up a proxy for it that
-runs in CloudHub.
+You can manage your API from API Manager to apply policies, add contracts, alerts, and SLA tiers. Additionally, you can configure a proxy for your API that runs in CloudHub. You can manage your API using the following steps:
```

Comment:
Proxy from an API can be deployed directly to CloudHub, RTF or using Hybrid. Also, they can be downloaded locally and uploaded/ deployed manually or through Studio. Can we explain a bit of it and then focus on CloudHub as the easiest/fastest option to have results?

Reply:
Done! Thanks! Though I feel we don't need to explain in detail in the getting started section.